src/Appstream/AppstreamClient.php

```diff
@@ -42,4 +42,16 @@
  * @method \Aws\Result updateStack(array $args = [])
  * @method \GuzzleHttp\Promise\Promise updateStackAsync(array $args = [])
  */
-class AppstreamClient extends AwsClient {}
+class AppstreamClient extends AwsClient {
+
+    public function __construct(array $args)
+    {
+        if (!isset($args['endpoint'])) {
```

Comment:
Yeah, that's true,, I also noticed the co-exist "appstream" and "appstream2", they cleared up the "appstream" 6 days ago, that's why changes are not synced. I try remove "appstream" and keep "appstream2" only in endpoint.json file. Yet it still doesn't work because we use service identifier.

Make sense, I'll make the `service` change instead.